pyhstr/application.py

```diff
@@ -2,13 +2,15 @@
 import os
 import re
 
-
 from pyhstr.user_interface import UserInterface
 from pyhstr.utilities import (
-    echo, read, remove_duplicates, sort, write
+    echo, detect_shell, read, read_ipython_history,
```

Comment:
```suggestion
    echo, detect_shell, read, get_ipython_history,
```

mypy would help with spotting errors like this